### PR TITLE
Duplicate -laws of chaos

### PR DESCRIPTION
**Pull request by patel-bhavin**
Code from https://github.com/splunk/security_content/pull/2534

### CODE REVIEW

File: detections/endpoint/powershell_fileless_script_contains_base64_encoded_content.yml [hunk 1]
```diff
@@ -45,32 +42,28 @@ tags:
...
-    - Victim
   product:
   - Splunk Enterprise
   - Splunk Enterprise Security
   - Splunk Cloud
   required_fields:
   - _time
   - ScriptBlockText
-  - Opcode
+  - Opcodepdetections/endpoint/powershell_fileless_script_contains_base64_encoded_content.yml
```
Comment (MHaggis) [inline detections/endpoint/powershell_fileless_script_contains_base64_encoded_content.yml]: typo?

Comment (TheLawsOfChaos) [reply detections/endpoint/powershell_fileless_script_contains_base64_encoded_content.yml]: 100%

Comment (patel-bhavin) [reply detections/endpoint/powershell_fileless_script_contains_base64_encoded_content.yml]: thank you !

**Comment by patel-bhavin** [comment]
Closing since we will merge the original

**Pull request closed without merging**